### ISSUE TITLE
[Bug]: Error when browsing models due to an early access model

### ISSUE THREAD

**Issue by Terroosh** [BlafKing/sd-civitai-browser-plus#99]
### Describe the bug.

While browsing models "Error" appears when I go to the next page. So I checked each model between the two pages with the civitai link and narrowed it down to the following model:

https://civitai.com/models/199189/ffusion-xl-lora-potion-art-engine

### Steps to reproduce the problem.

1. Add https://civitai.com/models/199189/ffusion-xl-lora-potion-art-engine
2. Press search
3. Error

### Expected behavior

Early Access models should be filtered out

### System info

* Extension version:1.15.1
* OS: Win 11
* SD-WebUI version:1.6
* Python:3.10.6

### Console logs

```Shell
Traceback (most recent call last):
  File "K:\SD A1111\venv\lib\site-packages\gradio\routes.py", line 488, in run_predict
    output = await app.get_blocks().process_api(
  File "K:\SD A1111\venv\lib\site-packages\gradio\blocks.py", line 1431, in process_api
    result = await self.call_function(
  File "K:\SD A1111\venv\lib\site-packages\gradio\blocks.py", line 1103, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "K:\SD A1111\venv\lib\site-packages\anyio\to_thread.py", line 33, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "K:\SD A1111\venv\lib\site-packages\anyio\_backends\_asyncio.py", line 877, in run_sync_in_worker_thread
    return await future
  File "K:\SD A1111\venv\lib\site-packages\anyio\_backends\_asyncio.py", line 807, in run
    result = context.run(func, *args)
  File "K:\SD A1111\venv\lib\site-packages\gradio\utils.py", line 707, in wrapper
    response = f(*args, **kwargs)
  File "K:\SD A1111\extensions\sd-civitai-browser-plus\scripts\civitai_api.py", line 545, in update_model_list
    HTML = model_list_html(gl.json_data, model_dict)
  File "K:\SD A1111\extensions\sd-civitai-browser-plus\scripts\civitai_api.py", line 242, in model_list_html
    updated_at = datetime.datetime.strptime(version['publishedAt'], "%Y-%m-%dT%H:%M:%S.%fZ")
TypeError: strptime() argument 1 must be str, not None
```

**Comment by AbsolomTDaak** [comment]
Maybe this note is related, if not please ignore, When looking at outdated models that need updating, it shows upcoming model versions which aren't released yet, is this a fault with civit ai website updating their information ahead of release, or something else.  Most notable on the Animesh and RunDiffusion XL checkpoints.  Just being curious, sorry for any inconvience

**Comment by BlafKing** [comment]
Thx for the bug report!

> it shows upcoming model versions which aren't released yet

That is indeed related! and no worries for any inconvience, appreciate you writing in!

This error occurs when a model has any versions that aren't published yet, I didn't take into account that a version could be returned in the API whilst not being published yet.
I'll adjust the code so that any versions that are not released yet will be hidden in the UI, which should solve this issue! :)

**Comment by AbsolomTDaak** [comment]
cheers for clarifying that, keep up the great work

**Comment by BlafKing** [comment]
This issue should now be fixed in the latest version ([v1.15.2](https://github.com/BlafKing/sd-civitai-browser-plus/commit/029cba905d19a4fd9f96ceb63a11918f46b3a1f7))

Thanks again for reporting it!
(Took a while since I was also working on making the default sub folders system work)
